extmod/modussl_axtls.c

```diff
@@ -167,10 +167,16 @@ STATIC mp_obj_ssl_socket_t *ussl_socket_new(mp_obj_t sock, struct ssl_args *args
         o->ssl_sock = ssl_client_new(o->ssl_ctx, (long)sock, NULL, 0, ext);
 
         if (args->do_handshake.u_bool) {
-            int res = ssl_handshake_status(o->ssl_sock);
-
-            if (res != SSL_OK) {
-                ussl_raise_error(res);
+            int r = ssl_handshake_status(o->ssl_sock);
+
+            if (r != SSL_OK) {
+                ssl_display_error(r);
```

Comment:
@tve I think this print was left in by accident?

Reply:
Honestly, I don't remember. I know that the axtls version prints errors currently and I believe I put that in to make it look the same as now. Also, I believe the errors are not very detailed and this prints more details. Probably best to remove and look at the tests output...

Reply:
I'll check it and probably just remove this line.

Reply:
This code does nothing because CONFIG_SSL_DIAGNOSTICS is disabled, so I'll just remove this.